src/datetimepicker.ios.js

```diff
@@ -29,71 +31,68 @@ const styles = StyleSheet.create({
   },
 });
 
-export default class Picker extends React.Component<IOSNativeProps> {
-  static defaultProps = {
-    mode: MODE_DATE,
-  };
+export default function Picker({
+  mode = MODE_DATE,
+  value,
+  locale,
+  maximumDate,
+  minimumDate,
+  style,
+  testID,
+  minuteInterval,
+  timeZoneOffsetInMinutes,
+  textColor,
+  onChange,
+  isDarkModeEnabled,
+}: IOSNativeProps) {
+  const dates: DatePickerOptions = {value, maximumDate, minimumDate};
+  const picker: NativeRef = React.createRef();
+  const textColorByDarkModeEnabled = isDarkModeEnabled
```

Comment:
this does not actually do anything - you just recreated the default behavior.

when dark mode is on, text is white,

when dark mode is off, text is black 

so this will not have any effect - what DOES have effect is `const backgroundColorByMode = isDarkModeEnabled ? Colors.black : Colors.white;` that you have added to `example/App.js`